### PR TITLE
codemod for top level imports

### PR DESCRIPTION
**Pull request by sri2k1us**

### CODE REVIEW

File: react-components/src/apps/admin/AppStats.js [hunk 1]
```diff
@@ -16,20 +16,22 @@ import {
...
+    Table,
+    TableBody,
+    TableCell,
+    TablePagination,
+    TableRow,
+    TableHead,
+    TableSortLabel,
+    Toolbar,
+    Toolbar as ToolbarGroup,
+    Toolbar as ToolbarSeparator,
```
Comment (ash3rz) [inline react-components/src/apps/admin/AppStats.js]: This seems like it's importing just the Toolbar 3 times, I think

File: react-components/src/notifications/view/NotificationToolbar.js [hunk 2]
```diff
@@ -10,18 +10,21 @@ import ids from "../ids";
...
 import RefreshIcon from "@material-ui/icons/Refresh";
 import DeleteIcon from "@material-ui/icons/Delete";
 import CheckIcon from "@material-ui/icons/Check";
 import intlData from "../messages";
-import Button from "@material-ui/core/Button";
-import { withStyles } from "@material-ui/core/styles";
+
+import {
+    Toolbar as ToolbarGroup,
+    Toolbar as ToolbarSeparator,
```
Comment (ash3rz) [inline react-components/src/notifications/view/NotificationToolbar.js]: Same potential problem here with the Toolbar getting imported twice

File: react-components/src/apps/admin/AppStats.js (hunk 1 above)
Comment (sri2k1us) [reply react-components/src/apps/admin/AppStats.js]: I found few issues with Styles as well

Comment (sri2k1us) [reply react-components/src/apps/admin/AppStats.js]: PS 👎 is for codemod not for your comments :)

File: react-components/src/notifications/view/NotificationToolbar.js (hunk 2 above)
Comment (sri2k1us) [reply react-components/src/notifications/view/NotificationToolbar.js]: I think `ToolbarGroup` and `ToolbarSeparator` is not supported anymore. I will go ahead and remove them.

File: react-components/src/apps/admin/AppStats.js (hunk 1 above)
Comment (ash3rz) [reply react-components/src/apps/admin/AppStats.js]: 😄 I figured, but thanks for the clarification

**Comment by ash3rz** [comment]
@sri2k1us Maybe it's time we close this PR? 😄

**Comment by sri2k1us** [comment]
> @sri2k1us Maybe it's time we close this PR? 😄

Nooo..I want to hold on it forever j/k

**Pull request closed without merging**